packages/expo-apple-authentication/README.md

```diff
@@ -17,8 +17,8 @@ For bare React Native projects, you must ensure that you have [installed and con
 
 ### Add the package to your npm dependencies
 
-```
-expo install expo-apple-authentication
+```sh
```

Comment:
what's the advantage of labeling this block? imo it's a bit cleaner to just leave this out.

Reply:
Visually, it doesn't give an advantage, We had been using it from some blocks in these readme files. I thought we might follow a pattern. I'll remove them.

Reply:
Removed ✅